mbedtls/Cargo.toml

```diff
@@ -52,7 +54,7 @@ cc = "1.0"
 
 [features]
 # Features are documented in the README
-default = ["std", "aesni", "time", "padlock"]
+default = ["std", "aesni", "time", "padlock", "dsa"]
```

Comment:
Let's not enable this by default. Also update README

Reply:
OK. I added `cargo test --features dsa` to `ct.sh` so tests still run in CI.